libraries/Microsoft.Bot.Builder/IUserTokenProvider.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
This interface can't possibly need all of these using statments. Could you remove & sort?

Reply:
fixed!